requirements.txt

```diff
@@ -1,36 +1,3 @@
-## Setup py requirements
-jsonpickle==1.0.0
-six==1.11.0
-redis==2.10.6
-fakeredis==0.10.3
-future==0.17.1
-numpy~=1.16.0
-typing==3.6.6
-ruamel.yaml==0.15.85
-requests~=2.2
-tensorflow==1.12.0
-h5py==2.7.1
-apscheduler==3.5.1
-tqdm==4.29.1
-ConfigArgParse==0.13.0
-networkx==2.2
-pykwalify==1.7.0
-coloredlogs==10.0
-flask==1.0.2
-flask_cors==3.0.7
-scikit-learn==0.20.2
-rasa_nlu~=0.14.6
-colorhash==1.0.2
-pika==0.12.0
-jsonschema==2.6.0
-packaging~=18.0
-gevent==1.4
-pytz==2018.9
-rasa-core-sdk~=0.13.1
-rasa_core~=0.13.8
-pymongo==3.7.2
-python-dateutil==2.7.5
-spacy~=2.0.18
-sklearn_crfsuite==0.3.6
-keras-applications==1.0.6
-keras-preprocessing==1.0.5
\ No newline at end of file
+numpy==1.16.4
+rasa
+spacy
```

Comment:
let's pin these rasa and spacy versions. the latest rasa and whatever spacy it pins should do it! also, we definitely don't need all of these requirements (I think the creator of this repo just copied over from rasa), but there are some that we will need. Off the top of my head, these `typing`, `requests` and `rasa-sdk`. Happy to help you look for more.

Sidenote, we should also change the imports in `actions.py` to import from `rasa_sdk` instead of `rasa_core_sdk`.

Reply:
I believe `typing` is part of the standard library. I've  checked all other imports in the repository and it seems  that `requests` and `rasa-sdk` are the only others requirements.

Reply:
awesome, thank you for going through those!

Reply:
@kearnsw what is numpy required for?

Reply:
This can be deleted now as well I believe.